Source/DiabloUI/diabloui.cpp

```diff
@@ -865,33 +888,36 @@ void Render(const UiList *uiList)
 
 void Render(const UiScrollbar *uiSb)
 {
+	const Surface out = Surface(DiabloUiSurface());

```

Comment:
@glebm was it intentional that this one is a copy while all other usages are references?

`const Surface &out = Surface(DiabloUiSurface());`

Reply:
It's an unowned surface, the other ones only work thanks to reference lifetime extension and should probably also be non-references (though it doesn't make a difference)